### ISSUE TITLE
При компиляции текущих исходников не работает главная страница

### ISSUE THREAD

**Issue by alxnovo** [tretyakovsa/Sonoff_WiFi_switch#34]
Если скомпилировать текущие исходники, то получаем вот такую главную страницу если переключиться на Jalousie 

![2017-09-26 13 09 36](https://user-images.githubusercontent.com/31793872/30855210-d86ef84a-a2bc-11e7-8ebb-a7d352d22788.png)

Через бинарники всё нормально работает, а вот через исходники нет

**Comment by renat2985** [comment]
Не стоит самому компилировать. Используйте готовые bin файлы. :) Просто на данный момент в исходниках еще не готовый код, мы внедряем новые функции. Как только мы их доделаем, протестируем, мы сразу выкладываем bin файлы в папку build. После чего у вас в ESP появляется красный треугольник. Так же все доступные сборки вы можете посмотреть в ESP на странице "Конфигурация".

**Comment by alxnovo** [comment]
Ну, мне всё-таки нужно немного подредактировать код (в жалюзях) для обесточивания всех реле через тайминг пока вы не реализовали в основном коде. 
Ладно пока буду использовать нестабильные исходники :)

**Comment by mikemdr** [comment]
>Не стоит самому компилировать. Используйте готовые bin файлы. :)
>"Компилируя исходники самостоятельно, вы как правило будите получать нестабильную сборку."
Следует ли это понимать как намёк на то, что проект является "как-бы закрытым" и исходники не соответствуют бинарникам, в коих в последствии (или уже?) могут появится "недокументированные фичи"?

**Comment by renat2985** [comment]
@mikemdr нет. Просто вся разработка ведется в master ветке. То есть как только мы более менее протестировали сборку мы выкладываем bin файлы. Вот в это время вы можете забирать исходники и компилировать самостоятельно. Только зачем? Далее мы снова начинаем добавлять новые функции изменяя код, после этого исходники могут быть не работоспособны, т.к. не всегда получается сразу сделать задуманное. Иногда исходники неделями лежать неработоспособными пока мы не придумаем как решить ту или иную задачу. Поэтому исходники качать не рекомендуется, у вас так же могут быть установлены не те библиотеки которые используем мы что тоже может повлиять на работу ESP.

**Comment by alexandersciv** [comment]
**@alxnovo**  Я столкнулся с подобным интерфейсом на смартфоне. В браузере штатном   подобная картина как у Вас, в Опера мини всё нормально. При этом в браузерах одинаковые настройки.

**Comment by mikemdr** [comment]
Под Win7 в IE, FireFox46 и ЯндексюБраузере - как у ТС.
